Desktop/Automation Home Tasks/HomeTask5/src/test/java/com/epam/automation/testsTestNG/MulCalcTest.java

```diff
@@ -0,0 +1,46 @@
+package com.epam.automation.testsTestNG;
+
+import com.epam.automation.BaseConfigurations.BaseTest;
+import org.testng.Assert;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+public class MulCalcTest extends BaseTest {
+
+
+    @Test(groups = "TestDivision", dataProvider = "DoubleValuesForMultiplication")
+    public void testMulOperationWithDoubleValues(double a, double b, double expected) {
+        double result = calculator.mult(a, b);
+        Assert.assertEquals(result, expected, "Incorrect result of multiplication operation with double params");
+    }
+
+
+    @DataProvider(name = "DoubleValuesForMultiplication")
+    Object[][] DoubleValuesForDivision() {
+        return new Object[][]{
+                {1.0, 1.0, 1.0},
+                {45.5, 1.0, 45.5},
```

Comment:
redundant cases. don't use multiple cases which have intention to caught the same issue.